streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskExecutor.java

```diff
@@ -185,12 +185,12 @@ void commitOffsetsOrTransaction(final Map<Task, Map<TopicPartition, OffsetAndMet
                     updateTaskCommitMetadata(allOffsets);
                 } catch (final TimeoutException timeoutException) {
                     log.error(
-                        String.format("Committing task(s) %s failed.",
+                        String.format("Committing task(s) %s failed due to %s",
                                       offsetsPerTask
                                           .keySet()
                                           .stream()
                                           .map(t -> t.id().toString())
-                                          .collect(Collectors.joining(", "))),
+                                          .collect(Collectors.joining(", ")), timeoutException.getMessage()),
                         timeoutException
```

Comment:
Not sure if I understand this change. Given that we pass `timeoutException` as second parameter into `log.error(...)` we should get the corresponding error message logged already?

Reply:
Nit: formatting:
```
log.error(
    String.format(
        "Committing task(s) %s failed due to %s",
        offsetsPerTask
            .keySet()
            .stream()
            .map(t -> t.id().toString())
            .collect(Collectors.joining(", ")),
        timeoutException.getMessage()
    ),
    timeoutException
);
timeoutException
```

Reply:
Yes, we pass `timeoutException` but If users truncate the stack trace that information is lost